clients/fides-js/src/components/Overlay.tsx

```diff
@@ -100,7 +109,15 @@ const Overlay: FunctionComponent<Props> = ({
     return () => clearTimeout(delayBanner);
   }, [setBannerIsOpen]);
 
+  const showModal = () => {
+    if (!modalLinkRef.current) {
+      document.body.classList.add("fides-overlay-modal-link-shown");
```

Comment:
Oh, I think I was confusing everyone about this.

I'd like us to add this class to the body immediately when the overlay renders, so that it's there and available as a utility class for the engineer to pull from.

So I'd do this right on line 120 instead, see next comment...